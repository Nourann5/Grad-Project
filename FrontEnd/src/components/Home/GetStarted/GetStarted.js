import React from 'react'
import { Col, Container, Row } from 'react-bootstrap'
import styles from './GetStarted.module.css'
import {ReactComponent as StarIcon} from 'assets/icons/star.svg'


function GetStarted() {
  return (
    <section id={styles["get-started"]}>
        <Container>
            <Row>
                
                <Col md='6' xs='12'>
                    <div className={styles["get-started__img"]}></div>
                </Col>
                <Col md='6' xs='12'>
                    <h2 className={styles["get-started__heading"]}>
                        HOW TO <span className={styles["get-started__heading-span"]}>GET STARTED</span> .. !
                    </h2>
                    <p className={styles["get-started__para"]}>
                        Lorem ipsum dolor sit amet, consetetur sadipscing Lorem ipsum dolor sit amet, consetetur sadipscing
                    </p>
                    <div className={styles["get-started__features"]}>
                        <div className={styles["get-started__feature"]}>
                            <div className={styles["get-started__icon"]}>
                                <StarIcon className={styles['get-started__icon-svg']}/>
                            </div>
                            <div className={styles["get-started__feature-desc"]}>
                                <h3 className={styles["get-started__feature-heading"]}>
                                HEADLINE
                                </h3>
                                <p className={styles["get-started__feature-para"]}>
                                Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                                </p>
                            </div>
                        </div>
                        <div className={styles["get-started__feature"]}>
                            <div className={styles["get-started__icon"]}>
                                <StarIcon className={styles['get-started__icon-svg']}/>
                            </div>
                            <div className={styles["get-started__feature-desc"]}>
                                <h3 className={styles["get-started__feature-heading"]}>
                                    HEADLINE
                                </h3>
                                <p className={styles["get-started__feature-para"]}>
                                    Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                                </p>
                            </div>
                        </div>
                        <div className={styles["get-started__feature"]}>
                            <div className={styles["get-started__icon"]}>
                                <StarIcon className={styles['get-started__icon-svg']}/>
                            </div>
                            <div className={styles["get-started__feature-desc"]}>
                                <h3 className={styles["get-started__feature-heading"]}>
                                    HEADLINE
                                </h3>
                                <p className={styles["get-started__feature-para"]}>
                                    Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                                </p>
                            </div>
                        </div>

                    </div>
                </Col>
            </Row>
        </Container>
    </section>
  )
}

export default GetStarted