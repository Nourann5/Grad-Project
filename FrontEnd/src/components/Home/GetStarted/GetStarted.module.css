
#get-started{
    background-color: var(--main-background-color);
    padding:40px 0;
}
.get-started__img{
    background: url('assets/imgs/started.png') no-repeat;
    background-size: contain;
    height: 100%;
    background-position: center;
}
.get-started__feature{
    display:flex;
}
.get-started__icon {
    width: 50px;
    display: flex;
    align-items: center;
    justify-content: center;
    color: #FFC850;
    margin-right: 20px;
}
.get-started__heading{
    color: black;
    font-size: 35px;
    font-family: 'PoppinsBold';
    margin-bottom: 20px;
}
.get-started__heading-span{
    color:#4807E6
}
.get-started__para{
    font-size:18px;
    font-family: 'PoppinsMedium';
    margin-bottom: 40px;
    color: black;
    line-height: 35px;
}
.get-started__feature-heading{
    font-family: 'PoppinsSemiBold';
    font-size: 20px;
    color: black;
    margin-bottom: 15px;
}
.get-started__feature-para{
    font-family: 'PoppinsMedium';
    color: black;
}
.get-started__feature{
    margin-bottom: 20px;
}
.get-started__icon-svg{
    --size:50px;
    width: var(--size);
    height: var(--size);
}
@media(max-width:992px){
    .get-started__img{
        height: 90%;
    }
}
@media(max-width:768px){
    .get-started__img{
        height: 600px;
        
    }
    .get-started__heading{
        text-align: center;
        margin-bottom: 15px
    }
    .get-started__para{
        font-size: 16px;
        text-align: center;
        margin-bottom: 30px;
    }
}

@media(max-width:426px){
    #get-started {
        padding: 0;
    }
    .get-started__features{
        width: 80%;
        margin: auto;
    }
    .get-started__feature-heading{
        font-size: 23px;
        margin-bottom: 5px;
    }
    .get-started__img {
        height:400px
    }
}
@media(max-width:376px){
    .get-started__features{
        width: 100%;
    }
    .get-started__feature-para{
        font-size:15px
    }
    .get-started__img {
        height:300px
    }
}
@media(max-width:376px){
    .get-started__feature-para{
        font-size:14px
    }
}