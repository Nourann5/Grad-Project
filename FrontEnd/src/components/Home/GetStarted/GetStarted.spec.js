import {render , screen} from '@testing-library/react'
import { BrowserRouter, Route } from 'react-router-dom'
import GetStarted from './GetStarted'

describe('Test the Home Gets Started', () => { 
    it('Heading exists',()=>{
        const tree =  render(
            <BrowserRouter>
                <GetStarted/>
            </BrowserRouter>
        )
        expect(tree).toMatchSnapshot()
    })
})