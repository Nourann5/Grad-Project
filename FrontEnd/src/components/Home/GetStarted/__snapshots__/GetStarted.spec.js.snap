// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Home Gets Started Heading exists 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <section
        id="get-started"
      >
        <div
          class="container"
        >
          <div
            class="row"
          >
            <div
              class="col-md-6 col-12"
            >
              <div
                class="get-started__img"
              />
            </div>
            <div
              class="col-md-6 col-12"
            >
              <h2
                class="get-started__heading"
              >
                HOW TO 
                <span
                  class="get-started__heading-span"
                >
                  GET STARTED
                </span>
                 .. !
              </h2>
              <p
                class="get-started__para"
              >
                Lorem ipsum dolor sit amet, consetetur sadipscing Lorem ipsum dolor sit amet, consetetur sadipscing
              </p>
              <div
                class="get-started__features"
              >
                <div
                  class="get-started__feature"
                >
                  <div
                    class="get-started__icon"
                  >
                    <svg
                      class="get-started__icon-svg"
                    >
                      star.svg
                    </svg>
                  </div>
                  <div
                    class="get-started__feature-desc"
                  >
                    <h3
                      class="get-started__feature-heading"
                    >
                      HEADLINE
                    </h3>
                    <p
                      class="get-started__feature-para"
                    >
                      Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                    </p>
                  </div>
                </div>
                <div
                  class="get-started__feature"
                >
                  <div
                    class="get-started__icon"
                  >
                    <svg
                      class="get-started__icon-svg"
                    >
                      star.svg
                    </svg>
                  </div>
                  <div
                    class="get-started__feature-desc"
                  >
                    <h3
                      class="get-started__feature-heading"
                    >
                      HEADLINE
                    </h3>
                    <p
                      class="get-started__feature-para"
                    >
                      Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                    </p>
                  </div>
                </div>
                <div
                  class="get-started__feature"
                >
                  <div
                    class="get-started__icon"
                  >
                    <svg
                      class="get-started__icon-svg"
                    >
                      star.svg
                    </svg>
                  </div>
                  <div
                    class="get-started__feature-desc"
                  >
                    <h3
                      class="get-started__feature-heading"
                    >
                      HEADLINE
                    </h3>
                    <p
                      class="get-started__feature-para"
                    >
                      Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </section>
    </div>
  </body>,
  "container": <div>
    <section
      id="get-started"
    >
      <div
        class="container"
      >
        <div
          class="row"
        >
          <div
            class="col-md-6 col-12"
          >
            <div
              class="get-started__img"
            />
          </div>
          <div
            class="col-md-6 col-12"
          >
            <h2
              class="get-started__heading"
            >
              HOW TO 
              <span
                class="get-started__heading-span"
              >
                GET STARTED
              </span>
               .. !
            </h2>
            <p
              class="get-started__para"
            >
              Lorem ipsum dolor sit amet, consetetur sadipscing Lorem ipsum dolor sit amet, consetetur sadipscing
            </p>
            <div
              class="get-started__features"
            >
              <div
                class="get-started__feature"
              >
                <div
                  class="get-started__icon"
                >
                  <svg
                    class="get-started__icon-svg"
                  >
                    star.svg
                  </svg>
                </div>
                <div
                  class="get-started__feature-desc"
                >
                  <h3
                    class="get-started__feature-heading"
                  >
                    HEADLINE
                  </h3>
                  <p
                    class="get-started__feature-para"
                  >
                    Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                  </p>
                </div>
              </div>
              <div
                class="get-started__feature"
              >
                <div
                  class="get-started__icon"
                >
                  <svg
                    class="get-started__icon-svg"
                  >
                    star.svg
                  </svg>
                </div>
                <div
                  class="get-started__feature-desc"
                >
                  <h3
                    class="get-started__feature-heading"
                  >
                    HEADLINE
                  </h3>
                  <p
                    class="get-started__feature-para"
                  >
                    Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                  </p>
                </div>
              </div>
              <div
                class="get-started__feature"
              >
                <div
                  class="get-started__icon"
                >
                  <svg
                    class="get-started__icon-svg"
                  >
                    star.svg
                  </svg>
                </div>
                <div
                  class="get-started__feature-desc"
                >
                  <h3
                    class="get-started__feature-heading"
                  >
                    HEADLINE
                  </h3>
                  <p
                    class="get-started__feature-para"
                  >
                    Lorem ipsum dolor sit amet, consetetur sadipscing Lorem
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
